gm4_better_fire/data/gm4_better_fire/loot_tables/flint.json

```diff
@@ -0,0 +1,55 @@
+{
```

Comment:
please put this under the namespace `/loot_tables/entities/flame_arrow` or something similar. Don't forget to update the functions that call this loot table